src/main/graphql/WorkflowJobGraphQL.graphql

```diff
@@ -0,0 +1,5 @@
+mutation WorkflowJobGraphQL($workflowId: Int!, $files: [FileInput]!) {
```

Comment:
I had to make a modification to the python client due to a bug, we were missing a flag here. Here the PR: https://github.com/IndicoDataSolutions/indico-client-python/pull/66/files

Reply:
Fixed